concourse-fly/README.md

```diff
@@ -0,0 +1,9 @@
+# Habitat package: concourse-fly
+## Maintainers
```

Comment:
This is a bit nitpicky so I feel bad even writing it but - My preference here would be to remove this "Maintainers" section and instead add yourself into the CODEOWNERS file just to keep with the pattern we've been using for other folks. That way you get appropriate notifications etc when changes have been submitted against this package.

Reply:
You're right, this is totally wrong. Amending.

Reply:
Think you've missed this one, didn't you? ;)